Project.toml

```
@@ -1,18 +1,23 @@
 name = "DiffResults"
 uuid = "163ba53b-c6d8-5494-b064-1a9d43ac40c5"
-version = "1.1.0"
+version = "1.0.3"
```

This line should not be reverted, I think:
```suggestion
version = "1.1.0"
```